x-pack/packages/ml/json_schemas/scripts/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { JsonSchemaService } from '../src/json_schema_service';
+
+const pathToOpenAPI = process.argv[2];
+
+new JsonSchemaService(pathToOpenAPI)
+  .createSchemaFiles()
+  .then(() => {
+    process.stdout.write('Schema files created successfully.');
+  })
+  .catch((e) => {
+    // TODO add kibana logger
```

Comment:
Is this something you plan to do for this PR?

Reply:
Done in https://github.com/elastic/kibana/pull/184518/commits/dbedeb0811325f1a44a85c00e08aeeea899fa46c